src/Syncromatics.Clients.WaySine/Models/Sign.cs

```diff
@@ -29,5 +29,8 @@ public DateTimeOffset UtcTimeStamp
         public int CellLinkStrength { get; set; }
         [JsonProperty("StopID")]
         public string StopId { get; set; }
+        public string StopName { get; set; }
+        public int? Rows { get; set; }
```

Comment:
technically, all the new signs from them will have these properties coming back but old signs exist where these will be null. I was on the fence whether this should be nullable or not

Reply:
I'm pro-nullable

Reply:
I spoke too soon.  This will be coming with all signs that use the WayStatus API (as opposed to the WayManager API)? In that case, maybe no nullable?

Reply:
good point, didnt realize that was the distinction, changing now